src/pyfaf/queries.py

```diff
@@ -1317,6 +1318,26 @@ def get_crashed_unknown_package_nevr_for_report(db, report_id):
             .all())
 
 
+def get_last_affected_version(db, report_id):
+    """
+    Return the last version of a CRASHED package in a report with a given report_id
+    """
+
+    (epoch, version, release) \
+    = (db.session.query(st.Build.epoch,
```

Comment:
I tried to run the code without doing `faf reposync` and got this error. 
```
ERROR:flask.app:Exception on /reports/5/ [GET]
Traceback (most recent call last):
  File "/usr/lib/python3.7/site-packages/flask/app.py", line 2292, in wsgi_app
    response = self.full_dispatch_request()
  File "/usr/lib/python3.7/site-packages/flask/app.py", line 1815, in full_dispatch_request
    rv = self.handle_user_exception(e)
  File "/usr/lib/python3.7/site-packages/flask/app.py", line 1718, in handle_user_exception
    reraise(exc_type, exc_value, tb)
  File "/usr/lib/python3.7/site-packages/flask/_compat.py", line 35, in reraise
    raise value
  File "/usr/lib/python3.7/site-packages/flask/app.py", line 1813, in full_dispatch_request
    rv = self.dispatch_request()
  File "/usr/lib/python3.7/site-packages/flask/app.py", line 1799, in dispatch_request
    return self.view_functions[rule.endpoint](**req.view_args)
  File "/usr/lib/python3.7/site-packages/webfaf/reports.py", line 576, in item
    forward['version'] = get_last_affected_version(db, report_id)
  File "/usr/lib/python3.7/site-packages/pyfaf/queries.py", line 1335, in get_last_affected_version
    .order_by(desc('epoch'), desc('version'), desc('release'))
TypeError: cannot unpack non-iterable NoneType object
```


What if the problem comes from unknown package? (`ReportUnknownPackage`)